package.json

```diff
@@ -22,21 +22,21 @@
   "license": "MIT",
   "version": "0.2.0",
   "engines": {
-    "node": ">=6.0.0"
+    "node": ">=8.9.0"
```

Comment:
What's the reason for this change?
Do updated dependencies require at least this specific version?

Reply:
i though it would be better to remove support for older version, but finally changed it back. Yet it can not be tested under node 7 and 9 (ava don't support it).
But it work on node 6.

Reply:
I guess it's not a big deal as they're both in _EOL_ state